REQUIRE

```diff
@@ -9,7 +9,7 @@ IterableTables 0.5
 TableTraitsUtils 0.1
 TableTraits
 DataValues
-#Widgets
-#Observables
+Widgets 0.4.0
```

Comment:
I think in principle if you don't use any new API developments in Widgets and Observables but simply need them to be 0.7 compliant you won't need to lower-bound them.

Reply:
In principle yes, but actually need some new `struct` that I've only defined recently (`AbstractObservable`) so I think it's safer to lower bound.